mkosi/resources/man/mkosi.1.md

```diff
@@ -1047,6 +1047,17 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 `KernelModulesInitrdExclude=`, `--kernel-modules-initrd-exclude=`
 :   Like `KernelModulesExclude=`, but applies to the kernel modules included in the kernel modules initrd.
 
+`FirmwareInclude=`, `--firmware-include=`
+:   Takes a list of regex patterns that specify firmware files to include in the image. Patterns should be
+    relative to `/usr/lib/firmware/<subdir>` paths. **mkosi** checks for a match anywhere in the firmware path
+    (e.g. `bcm8483` will match against `cxgb4/bcm8483.bin`). All firmware files that match any of the specified
+    patterns are included in the image.
+
+`FirmwareExclude=`, `--firmware-exclude=`
```

Comment:
We should mention here that firmware is excluded even if an included module depends on it.